apps/cookbook/src/app/examples/slides-example/examples/custom-heading.ts

```diff
@@ -4,8 +4,8 @@ import { CardModule } from '@kirbydesign/designsystem/card';
 
 const config = {
   selector: 'cookbook-slides-custom-heading-example',
-  template: `<kirby-slides [slides]="slides" [title]="'Title'" [showNavigation]="true">    
-  <h3>Custom heading</h3>
+  template: `<kirby-slides [slides]="slides" [showNavigation]="true">    
+  <h1>Custom heading</h1>
```

Comment:
Consider preserving a level that makes sense for the surrounding heading levels in the cookbook.
```suggestion
  <h3>Custom heading</h3>
```